airflow/providers/amazon/aws/hooks/eks.py

```diff
@@ -35,6 +35,7 @@
 
 DEFAULT_PAGINATION_TOKEN = ''
 STS_TOKEN_EXPIRES_IN = 60
+AUTHENTICATION_API_VERSION = "client.authentication.k8s.io/v1alpha1"
```

Comment:
Should we use `client.authentication.k8s.io/v1beta1`?

Reply:
I believe we went with alpha since that's what the [EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html) still shows, but I reckon beta will work just fine.

Whichever you go with should probably be the same as what you use in `eks_get_token.py `

Reply:
Niko is correct, I'm not sure what the functional difference is, but the docs said (and still say) alpha so that is what I went with in the initial contribution.  That doesn't mean we can't go beta, just explaining why I went with what I did.

Reply:
Sounds good!